engine/battle/core.asm

```diff
@@ -3715,23 +3715,23 @@ TryToRunAwayFromBattle:
 	inc a
 	ld [wNumFleeAttempts], a
 	ld a, [hli]
-	ldh [hPartyMon1Speed + 0], a
+	ldh [hMultiplicand + 1], a
 	ld a, [hl]
-	ldh [hPartyMon1Speed + 1], a
+	ldh [hMultiplicand + 2], a
 	ld a, [de]
 	inc de
 	ldh [hEnemyMonSpeed + 0], a
 	ld a, [de]
 	ldh [hEnemyMonSpeed + 1], a
 	call Call_LoadTempTileMapToTileMap
-	ld de, hPartyMon1Speed
+	ld de, hMultiplicand + 1
```

Comment:
Are you really sure about this one? It really isn't used as a multiplicand at this point, though I see a `call Multiply` a little further below.

Reply:
Yes. The `call Multiply` at line 3737 only initializes `hMultiplicand + 0` because `+ 1` and `+ 2` were initialized with the party mon speed. And `hMultiplier` is 32, which makes sense given the [escape chance formula](https://bulbapedia.bulbagarden.net/wiki/Escape#Generation_I_and_II), which is just what this block is doing.

Reply:
what's the point of `+ 0`?

Reply:
Lots of multi-byte buffers use `+ 0` for consistency with nearby `+ 1`, `+ 2`, etc, and to avoid implying that `[wSomeBuffer]` is a useful single-byte quantity. Although we probably haven't applied this convention everywhere.

Reply:
It seems rather obscure to me, as the first reaction to seeing something like `+ 0` is "what is this even for?". It's similar to the old `+- 1` in that regard; its meaning isn't really obvious.

It reminds me of some SQL versions where `foo` and `+foo` behaved differently.